editor/ok_interface.py

```diff
@@ -93,116 +30,167 @@ def write(self, message):
     def flush(self):
         sys.__stdout__.flush()
 
-
-def run_tests():
-    LOGGING_FORMAT = '%(levelname)s  | %(filename)s:%(lineno)d | %(message)s'
-    logging.basicConfig(format=LOGGING_FORMAT)
-    log = logging.getLogger('client')  # Get top-level logger
-
-    # noinspection PyUnresolvedReferences
-    from client.cli.ok import parse_input
-    log.setLevel(logging.ERROR)
-
-    args = parse_input(["--all", "--verbose", "--local"])
-
+def capture_output(console, lines):
     old_stdout = sys.stdout
     sys.stdout = out = PrintCapture()
+    result = console._interpret_lines(lines)
+    sys.stdout = old_stdout
+    return result, out.log
 
-    assign = assignment.load_assignment(None, args)
+class PromptOutput(metaclass=ABCMeta):
+    @abstractmethod
+    def represenation(self):
```

Comment:
Should be "representation", right?

Same change for all the classes implementing it.